openjdk-7.yaml

```diff
@@ -0,0 +1,281 @@
+package:
+  name: openjdk-7
```

Comment:
should we put `unsupported` or `bootstrap` in the name here to avoid folks using it without realizing it's not getting security updates?

Reply:
it's in the description.  i'd rather not rename it, because in the near future it will be moved to a special repo for bootstrap components.  that repo just doesn't presently exist.